frontend/src/pages/nft/nft.tsx

```diff
@@ -13,9 +13,16 @@ import { getIpfsLink } from '@/utils';
 import { ListingCard } from './components';
 import { useNFT } from './hooks';
 import InfoSVG from './info.svg?react';
+import FeaturedPlayListSVG from './featured-play-list.svg?react';
+import MarkerSVG from './marker.svg?react';
 import styles from './nft.module.scss';
+import { useState } from 'react';
```

Comment:
Import's order is configured with project's lint rules, libraries imports should have first priority.